test/views/declaration/PreviousDocumentsViewSpec.scala

```diff
@@ -113,6 +113,14 @@ class PreviousDocumentsViewSpec extends ViewSpec with PreviousDocumentsMessages
       val saveButton = getElementByCss(view, "#submit")
       saveButton.text() must be(messages(saveAndContinueCaption))
     }
+
+    "display 'Save and return' button on page" in {
+
+      val view = createView()
+
+      val saveAndReturnButton = getElementByCss(view, "#submit_and_return")
```

Comment:
getElementByCss is deprecated so we probably shouldnt add more

This can be replaced by `view.getElementById("submit_and_return")`

Reply:
very very good point! Sure! It will be the last thing I will do ;)

Reply:
this has been addressed